tests/blueprints/test_cherrypicked_plates.py

```diff
@@ -44,13 +48,16 @@ def test_get_cherrypicked_plates_endpoint_no_barcode_in_request(
     assert response.json == {"errors": ["GET request needs 'barcode' in url"]}
 
 
-def test_get_cherrypicked_plates_endpoint_no_positive_samples(app, client):
+def test_get_cherrypicked_plates_endpoint_no_robot_number_in_request(
+    app, client, samples_with_lab_id
+):
     response = client.get(
         "/cherrypicked-plates/create?barcode=plate_1",
         content_type="application/json",
     )
+
     assert response.status_code == HTTPStatus.BAD_REQUEST
-    assert response.json == {"errors": ["No samples for this barcode: plate_1"]}
+    assert response.json == {"errors": ["GET request needs 'robot' in url"]}
```

Comment:
I think checking the content of the error might be a bit too restrictive on future changes. Could you instead check that the length of the errors value array is 1?

Reply:
But depends on the situation; sometimes we want to check the user is receiving a valid error message that has some description of the problem.

Reply:
I'm always inclined to not check non-data driven strings in tests, as they can be changed without changing the functionality of the method, but I'll let you decide the best approach here, as I'm not sure there is a clear-cut "right answer" 🙂